.circleci/config.yml

```diff
@@ -759,14 +759,14 @@ workflows:
           filters:
             branches:
               only:
-                - develop
+                - fix/cocoapods-upgrade
```

Comment:
this needs to be reverted back to develop branch in order for the pipeline to work.

Reply:
@antonioned thanks mate will update now